srtcore/buffer_snd.h

```diff
@@ -130,14 +134,29 @@ class CSndBuffer
     SRT_ATTR_EXCLUDES(m_BufLock)
     time_point peekNextOriginal() const;
 
+    struct Drop
+    {
+        static const size_t BEGIN = 0, END = 1;
+        int32_t seqno[2];
```

Comment:
Maybe separate variables would make the code easier to read.
```
int32_t iSeqnoFirst;
int32_t iSeqnoLast; // Including this seqno?
```

Reply:
The array is here for a reason - it's used for sending data in a control packet. Otherwise I'd have to create an array there on the fly and fill in with the data read from such fields.

Reply:
Right, missed that. 👍